#!/usr/bin/env python
PACKAGE = "velodyne16"

from math import pi
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_range", double_t, 0, "min range to publish", 0.9, 0.1, 10.0)
gen.add("max_range", double_t, 0, "max range to publish", 130, 0.1, 200)
gen.add("velodyne_static", bool_t, 0, "whether to use tf",  True)
gen.add("fixed_frame_id", str_t, 0, "fixed frame for point clouds", "world")

exit(gen.generate(PACKAGE, "cloud_vlp16_node", "CloudVLP16Node"))
